### DISCUSSION TITLE
problem running with MSVC 2017

### DISCUSSION THREAD

**Discussion by jbensabat** [alandefreitas/matplotplusplus#147] [General]
Hello 
I managed building the lib under MSVC 2017 and CMake GUI.
I also installed gnuplot and added the path
when I try to run the first example I get the following message 

gnuplot' is not recognized as an internal or external command,
operable program or batch file.

please help 
best
jac

**Comment by lacc97** [comment]
Are you sure you've correctly installed gnuplot? Can you run gnuplot from a cmd shell or Powershell? This seems like an issue with your PATH.

**Comment by jbensabat** [comment]
Hi
I managed to visualize the graph opening gnuplot in a cmd shell
but the graph is frozen (no possibility to move the window and or close it even)
is there a way to draw independently of a gnuplot shell ?
best
jac
